docs/semgrep-secrets/policies.md

```diff
@@ -0,0 +1,119 @@
+| Severities | Filter by the severity level of the secret |
+| Analysis method | Filter based on whether **Semantic** or **Generic** analysis |
+
+## Rule entry reference
+
+This section defines the columns of the rule entries in the Policies page:
+
+| Filter | Description |
+| -------  | ------ |
+| Rule name  | Name of the rule Semgrep Secret uses for scanning. |
+| Labels  | Metadata describing the rule, including the service for which the rule is applicable. |
+| Open findings  | The number of open findings the rule detected across all scans.  |
+| Fix rate  | The percentage of findings that are fixed.  |
+| Severity  | The higher the severity, the more critical the issues that a rule detects.      |
+| Confidence  | Indicates confidence of the rule to detect true positives.      |
+| Source  | Indicates the origin of a rule. | <ul><li><strong>Pro:</strong> Authored by Semgrep. Custom:</strong> Rules created within your Semgrep organization. |
+| Ruleset  | Rules are also organized in rulesets. Currently, only `semgrep-secrets` is available. |
+| Mode  | Specifies what workflow action Semgrep performs when a rule detects a finding. An additional filter, **Disabled**, is provided for rules you have turned off and are no longer included for scanning. | See [Rule modes](#rule-modes) documentation. |
+
+## Rule modes
+
+Semgrep Secrets provides three rule modes. These can be used to trigger **workflow options** whenever Semgrep Secrets identifies a finding based on the rule:
+
+| Rule mode | Description |
+| -------   | ------ |
+| Monitor   | Rules in **Monitor mode** display findings only in: <ul><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>Set rules to this mode to evaluate their true positive rate and other criteria you may have. By keeping rules in Monitor, developers do not receive potentially noisy findings in their PRs or MRs.  |
+| Comment   | Rules in **Comment mode** display findings in:<ul><li>Developers' PRs or MRs</li><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>Set rules that have met your performance criteria to this mode when you are ready to display findings to developers.     |
+| Block     | Rules in **Block mode** cause the scan job to fail with an exit code of `1` if Semgrep Secrets detects a finding from these rules. You can use this result to enforce a block on the PR or MR. For example, GitHub users can enable branch protection and set the PR to fail if the Semgrep step fails. <br />These rules display findings in:<ul><li>Developers' PRs or MRs</li><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>These are typically high-confidence, high-severity rules. |
+
+## Validation state policies
+
+You can define how Semgrep handles findings that it categorizes as invalid or results in a validation error.
+
+- **Invalid findings**: include secrets that, during validation, were identified as never functional or revoked.
```

Comment:
```suggestion
- **Invalid findings**: include secrets that, during validation, were identified as revoked, or were never functional.
```
The original:

"...were identified as never functional or revoked" can be interpreted in 2 ways:
- never functional and never revoked
- never functional and revoked (the correct interpretation)

So I reworded such that "never" is not in danger of implicitly modifying "revoked"